index.js

```diff
@@ -8,11 +8,11 @@ app.get('/getWeather', function(req, res){
     axios.get(url)
         .then((response) => {
             console.log(response.data)
-            res.status(200).json({"Temperature is: ": response.data + " F"})
+            res.status(200).json({"Temp: ": response.data})
```

Comment:
You don't need the colon included in the key. Only the one in the middle, separating the key and value.